### DISCUSSION TITLE
Is SignalR still on the roadmap?

### DISCUSSION THREAD

**Discussion by dylan-mills** [Azure/azure-mobile-apps#579] [Q&A] [answered]
Hi, I was listening to your talk on the .NET Data Community standup from September 2022 and at the end you mentioned adding in SignalR for more real-time-ish changes.  Just curious if this still planned?

https://youtu.be/FbV4VWf7PfQ?t=3428

Thanks for all your great work on this!

**Comment by adrianhall** [comment, marked as answer]
Still on the roadmap (I should really add an issue for it).  Still not started.
